astrokat/noisediode.py

```diff
@@ -175,6 +175,29 @@ def _nd_log_msg_(ant,
     return actual_time
 
 
+def nd_reset(kat,
+             timestamp,
```

Comment:
You say in the docstring that this is optional but there is no default value... I would just make the default value "now".